scene/resources/3d/ik_constraint_3d.cpp

```diff
@@ -130,9 +133,50 @@ void IKConstraint3D::_bind_methods() {
 	ADD_ARRAY_COUNT("Holes", "hole_count", "set_hole_count", "get_hole_count", "holes/");
```

Comment:
```suggestion
	ADD_ARRAY_COUNT("holes", "hole_count", "set_hole_count", "get_hole_count", "holes/");
```

Godot uses Humanize in with variable names.